src/components/LessonList/components/PaginatedLessonList/components/LessonListItem/index.js

```diff
@@ -4,8 +4,9 @@ import {Heading, Markdown} from 'egghead-ui'
 import LessonState from 'components/LessonState'
 import LessonActions from 'components/LessonActions'
 
-export default ({instructor, lesson}) => {
-
+export default ({lesson}) => {
```

Comment:
this component is only used in the one place and the instructor isn't passed in. I guess that might technically be the "instructor logged in"?